pkg_resources/__init__.py

```diff
@@ -126,6 +133,10 @@ class _LoaderProtocol(Protocol):
     def load_module(self, fullname: str, /) -> types.ModuleType: ...
 
 
+class ZipLoaderModule(Protocol):
```

Comment:
If we are introducing a new name here, better to make explicitly private, as not everyone pays attention to `__all__` (unfortunately Hyrum's law hits hard on setuptools/pkg_resources).